tensorflow/contrib/layers/python/layers/layers.py

```diff
@@ -914,8 +912,9 @@ def layer_norm(inputs,
     # Calculate the moments on the last axis (layer activations).
     mean, variance = nn.moments(inputs, axis, keep_dims=True)
     # Compute layer normalization using the batch_normalization function.
+    variance_epsilon = 1E-12
```

Comment:
@sguada I had originally set this to 0.001 but receive feedback that it is too large. I'm not sure what the correct value to put here is. Do you have a suggestion?